dashboard/dashboard-overview.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Overview Page
+summary: Learn the overview page of TiDB Dashboard.
+category: how-to
+---
+
+# Overview Page
+
+This page shows the overview of the entire TiDB cluster, including the following information:
+
+- Queries per second (QPS) of the entire cluster.
+- The query latency of the entire cluster.
+- The SQL statements that have cumulated the longest execution time over the recent period.
+- The slow queries whose execution time over the recent period exceeds the threshold.
+- The node count and status of each instance.
+- Monitor and alert messages.
+
+## Enter the page
+
+After logging into TiDB Dashboard, the overview page is entered by default, or you can click **Overview** on the left navigation bar to enter this page:
+
+![Enter overview page](/media/dashboard/dashboard-overview-access.png)
+
+## QPS
+
+This area shows the number of successful and failed queries per second for the entire cluster over the recent hour:
+
+![QPS](/media/dashboard/dashboard-overview-qps.png)
+
+> **Note:**
+>
+> This feature is available only in the cluster where the Prometheus monitoring component is deployed. If Prometheus is not deployed, there will be a display failure.
+
+## Latency
+
+This area shows the latency of 99.9%, 99%, and 90% of queries in the entire cluster over the recent one hour:
+
+![Latency](/media/dashboard/dashboard-overview-latency.png)
+
+> **Note:**
+>
+> This feature is available only on the cluster where the Prometheus monitoring component is deployed. If Prometheus is not deployed, there will be a display failure.
+
+## Top SQL statements
+
+This area shows the ten types of SQL statements that have accumulated the longest execution time in the entire cluster over the recent period. SQL statements with different query parameters but of the same structure are classified into the same SQl type of and displayed in the same row:
```

Comment:
`same SQl type of and` --> `same SQL type`?

Reply:
Still remain the `SQl` which should be `SQL`.

Reply:
Fixed.